service/csi_extension_server.go

```diff
@@ -294,19 +296,20 @@ func (s *service) checkIdempotency(ctx context.Context, snapshotsToMake *siotype
 	//go through the existing vols, and update maps as needed
 	//check that all idempotent snapshots belong to the same consistency group.
 	//this check verifies criteria #2
-	consitencyGroupValue := snapGrpID
+	consitencyGroupValue := ""
```

Comment:
Could we take the chance here to fix the spelling on this variable? Realize it's not part of the changes here but it would be good to take care of.

Reply:
fixed